docs/manuals/source/Appendix/BareosPrograms.rst

```diff
@@ -789,6 +789,20 @@ After bscan
 If you use bscan to enter the contents of the Volume into an existing catalog, you should be aware that the records you entered may be immediately pruned during the next job, particularly if the Volume is very old or had been previously purged. To avoid this, after running bscan, you can manually set the volume status (VolStatus) to Read-Only by using the update command in the catalog. This will allow you to restore from the volume without having it immediately purged. When you have restored and
 backed up the data, you can reset the VolStatus to Used and the Volume will be purged from the catalog.
 
+.. limitation:: bscan: logical job order after bscan
```

Comment:
```suggestion
.. limitation:: importing jobs with bscan may lead to incorrect order in the job chain
```
This is my poor attempt to merge your last sentence into the title.

Reply:
Changed the limitation title.